contrib/demo-rgb.sh

```diff
@@ -158,3 +158,34 @@ btc-cold finalize --publish testnet ${PSBT}
 
 # Now, once the transaction will be mined, the verification should pass
 rgb consignment validate ${CONSIGNMENT}
+
+# -- CONSUME AND UNLOCK ASSET ------------------------------------------------
+# Go to a remote server / other machine and do the following
+
+# First, we must get all the information required to consume the consignment file.
+# The consignment file
+CONSIGNMENT=${DIR}/demo.rgbc
+# The UXTO used in blinded utxo operation.
+RECEIVE_UXTO=$UTXO
+# The blinding factor generated in blinded utxo operation.
+BLINDING_FACTOR=$INVOICE_BLINDING
+# The close method used in PAYMENT operation.
+CLOSE_METHOD="tapret1st"
+
+# Next, we need to compose the reveal information to unlock the uxto.
+REVEAL="$CLOSE_METHOD@$RECEIVE_UXTO#$BLINDING_FACTOR"
+
+# Let's consume and reveal the concealed seal inside the consignment file.
+rgb -n testnet transfer consume ${CONSIGNMENT} --reveal ${REVEAL}
```

Comment:
The command  is `rgb-cli`

Reply:
Yes, you are correct!

I will change it!

Thanks!